dsb-integration/src/test/java/com/spectralogic/dsbrowser/integration/tasks/Ds3PutJobTest.java

```diff
@@ -103,14 +101,18 @@ public static void setConnection() {
             session = CreateConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
 
             final Path path;
+            ImmutableList<Pair<String,Path>> pair;
             try {
                 path = ResourceUtils.loadFileResource("files/SampleFiles.txt");
                 file = path.toFile();
+                pair = ImmutableList.of(new Pair<String,Path>("files/SampleFiles.txt", path));
             } catch (final URISyntaxException | FileNotFoundException e) {
+                pair = null;
                 e.printStackTrace();
```

Comment:
If we want to fail when the exception is thrown, just throw it, and let the outer test block catch it.